src/frontend/apps/standalone_site/src/features/Header/Header.tsx

```diff
@@ -76,10 +128,58 @@ const Header = forwardRef<Nullable<HTMLDivElement>>((_props, ref) => {
         <Burger width={60} height={60} aria-controls="menu" />
         <LogoIcon width={117} height={80} />
       </Box>
-      <Box direction="row" align="center" gap="small" justify="end">
-        <Text>{fullName}</Text>
-        <AvatarIcon width={42} height={42} />
-      </Box>
+
+      <DropButton
+        plain
+        label={
+          <Box direction="row" align="center" gap="small" justify="end">
+            <Text>{fullName}</Text>
+            <AvatarIcon width={42} height={42} />
+          </Box>
+        }
+        dropAlign={{ top: 'bottom', right: 'right' }}
+        dropContent={
+          <Box direction="column" margin="small" gap="small">
+            <NavLinkStyled to={routes.PROFILE.path}>
+              {intl.formatMessage(messages.profile)}
+            </NavLinkStyled>
+            <NavLinkStyled to={routes.PROFILE_SETTINGS.path}>
+              {intl.formatMessage(messages.settings)}
+            </NavLinkStyled>
+            <ButtonStyled
+              plain
+              onClick={() => {
+                const performLogout = async () => {
+                  await fetch('/account/api/logout/', {
```

Comment:
Maybe better in `features/Authentication/api`, what do you think?

Reply:
Huum, it's not really related to the authentication imo, and I thought a feature should not use something from an other feature unless it's a parent one.

Reply:
No you can of course, the system should stay a minimum flexible and DRY, however you have to expose what you need out of the feature in index.
Seems for me to be more in authentication than in header, if you need another logout button in the footer by example, seems strange to get it expose from the header. Up to you.